### PR TITLE
MRF-74112/snapshot-version-fix

### PR DESCRIPTION
**Pull request by yusufyilmazmrf**

### CODE REVIEW

File: utils/index.js [hunk 1]
```diff
@@ -1,19 +1,17 @@
 const { execSync } = require('child_process');
 
-function createSnapShotVersion() {
-	const { version } = require(`${process.cwd()}/package.json`);
+function createSnapShotTag() {
```
Comment (joselanderosmrf) [inline utils/index.js]: do you still use this?

Comment (yusufyilmazmrf) [reply utils/index.js]: deleted 👍

File: providers/widget/publish/main.js [hunk 2]
```diff
@@ -15,14 +14,18 @@ try {
             `npx provider-cli docs:publish --gh-token ${ghToken}`
         ],
         '🚀Publishing widget 🕹Catalog🕹...'
-    );
+	);
+	
+	utils.execStep(
+		`npm i npm-snapshot -g`
```
Comment (joselanderosmrf) [inline providers/widget/publish/main.js]: since you use npx you dont have to install any more

Comment (yusufyilmazmrf) [reply providers/widget/publish/main.js]: ahh i wouldnt know that. That's nice 👍

**Review by joselanderosmrf — APPROVED** [review]

**Pull request merged**